Language versions (#29)
inbo/protocols: CONTRIBUTING.md

1. make sure your local clone of the remote repository is up to date:
    1. with the master branch checked out, press the pull button in the Git pane

> 1. a subject-matter specialist uses `protocolhelper::create_protocol()` to start a new [protocol from a template](#from-a-new-template)

Reviewer: ```suggestion
1. a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to start a new [protocol from a template](#from-a-new-template)
```

For an **update** of an existing protocol all steps are the same, except for:

- you don't need `protocolhelper::create_protocol()`
- the creation of the new branch can (re-)use the protocol-code of the existing protocol
- after review is finished, the protocol-specific `NEWS.Rmd` should be updated to document the substantive changes between the updated version of the previous version.

For adding a **pre-existing version of a protocol that was written in `docx` format**, follow the steps to create a new protocol, except in the second step:

> - a subject-matter specialist uses `protocolhelper::create_protocol()` to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).

Reviewer: ```suggestion
- a subject-matter specialist uses `protocolhelper::create_protocol()` (or one of its shortcut functions `create_sfp()` or `create_spp()`) to convert the `docx` protocol to Rmarkdown files. See section [From an existing docx protocol](#from-an-existing-docx-protocol).
```

           render = FALSE)
```

Alternatively, for a project-specific protocol:

```r
library(protocolhelper)
create_spp(title = "Bodemstalen nemen", 
           subtitle = "", 
           short_title = "bodemstalen", 
           authors = c("Jon Beton", "Jef Plastiek"), 
           date = Sys.Date(), 
           reviewers = c("reviewer1, reviewer2"), 
           file_manager = "manager",
           project_name = "mne",
           language = "nl",
           version_number = "2021.01",
           render = FALSE)
```

> For project-specific protocols, `protocolhelper::add_subprotocol()` can be used subsequently to add (sections of) published standard field protocols as subprotocols.

Reviewer: Best add that a specific version needs to be set for each subprotocol (it should not evolve dynamically within a project protocol version). Optionally, project-specific parameters can be passed.

## Protocol language

Protocols can be written in either one or multiple languages. 
The functions in the `protocolhelper` package support Dutch (language = 'nl') and English (language = 'en').

> A translation of the same protocol will be evident from the protocol-code, e.g.: `sfp-001-nl` and `sfp-001-en`.

Reviewer: ```suggestion
A translation of the same protocol will be evident from the protocol-code, e.g.: `sfp-001-nl` and `sfp-001-en`.
Note that these must be considered as just two different protocols (with a different protocol code), although the protocol prefix+number shows their relationship.
```

> To indicate that `sfp-001-en` is a literal translation of `sfp-001-nl`, the protocol-specific `NEWS.Rmd` file should mention this together with the version number of `sfp-001-nl`.

Reviewer: ```suggestion
To indicate that a specific version of `sfp-001-en` is a literal translation of a version of `sfp-001-nl`, the `NEWS.Rmd` file of `sfp-001-en` should mention this, in a version-specific way.
```

> Different language versions of the same protocol should be added to the repo in a separate pull request.

Reviewer: ```suggestion
Different language variants of the 'same' protocol should be added to the repo in a separate pull request.
```

> Note that in subsequent versions of - say - `sfp-001-en` it is allowed that it's contents diverges from the version of `sfp-001-nl` of which it was a literal translation.

Reviewer: ```suggestion
Note that in subsequent versions of - say - `sfp-001-en` it is allowed that its contents diverge from the version of `sfp-001-nl` of which it was a literal translation.
```
